src/client/clientmap.cpp

```diff
@@ -715,7 +715,6 @@ void ClientMap::updateDrawList()
 		g_profiler->avg("MapBlocks sides skipped [#]", sides_skipped);
 		g_profiler->avg("MapBlocks examined [#]", blocks_visited);
 	}
-	g_profiler->avg("MapBlocks shortlist [#]", shortlist.size());
```

Comment:
I recently found it useful to see the size of the shortlist, to explain why `updateDrawList` takes less time with larger settings of `client_mesh_chunk`.

Reply:
that sounds like a code bug actually.

Reply:
Yes... For the normal culler we do not need the shortlist in the first place.
And if the loop culler is enabled `touchMapBlocks` is a noop.